### PR TITLE
feat(data-table-manager): add prop `managerTheme`

### PR DESCRIPTION
**Pull request by jonnybel**
#### Summary

This PR adds a prop `settingsContainerTheme` which allows changing the background colour (theme) of the Card which contains the manager settings.

### CODE REVIEW

**Review by tdeekens — APPROVED** [review]
My main/only concern is with the name containerTheme. It sort of implies I can in the future combine that theme with something else. Like containerTheme and dropdownTheme. Which I think will never be true not meaningful (might be wrong). As a result maybe just theme is enough as a name.

File: packages/components/data-table-manager/src/data-table-manager.js [hunk 1]
```diff
@@ -234,6 +235,13 @@ DataTableManager.propTypes = {
    * A React node for rendering additional information within the table manager.
    */
   topBar: PropTypes.node,
+  /**
+   * Sets the background theme of the Card that contains the settings
+   */
+  settingsContainerTheme: PropTypes.oneOf(['light', 'dark']).isRequired,
```
Comment (emmenko) [inline packages/components/data-table-manager/src/data-table-manager.js:241]:
```suggestion
  settingsContainerTheme: PropTypes.oneOf(['light', 'dark']),
```

File: packages/components/data-table-manager/src/settings-container/settings-container.js [hunk 2]
```diff
@@ -73,6 +69,10 @@ SettingsContainer.propTypes = {
   primaryButton: PropTypes.element,
   secondaryButton: PropTypes.element,
   children: PropTypes.node.isRequired,
+  containerTheme: PropTypes.oneOf(['light', 'dark']).isRequired,
```
Comment (emmenko) [inline packages/components/data-table-manager/src/settings-container/settings-container.js:72]:
```suggestion
  containerTheme: PropTypes.oneOf(['light', 'dark']),
```

**Review by emmenko — APPROVED** [review]

**Comment by jonnybel** [comment]
> My main/only concern is with the name containerTheme. It sort of implies I can in the future combine that theme with something else. Like containerTheme and dropdownTheme. Which I think will never be true not meaningful (might be wrong). As a result maybe just theme is enough as a name.

How about changing it to `managerTheme`? I didn't want to call it "theme" because we're using that prop name for making components _themeable_.

**Comment by tdeekens** [comment]
Yeah, managerTheme is fine. That indicates what behaviour is themed not the element.

**Review by tdeekens — APPROVED** [review]

**Pull request merged**